djangocms_page_sitemap/cms_toolbars.py

```diff
@@ -44,8 +41,6 @@ def populate(self):
             except AttributeError:
                 not_edit_mode = not self.toolbar.edit_mode_active
             current_page_menu = self.toolbar.get_or_create_menu('page')
-            position = current_page_menu.find_first(
```

Comment:
@vipulnarang95 Why has this been removed, the menu is still valid for Vanilla CMS4

Reply:
This was not allowing to add the Menu at that level.